setup.py

```diff
@@ -47,7 +47,7 @@ def readme():
         'httplib2',
         'jinja2',
         'pyrabbit==1.1.0',
-        'urllib3==1.23',
+        'urllib3>=1.24.2',
```

Comment:
Also this should be `urllib3==1.24.2`, or it will break all the functional tests.